libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql

```diff
@@ -37,13 +37,16 @@ query ClientProfiles(
   clientProfiles(filters: $filters, pagination: $pagination, order: $order) {
     ... on ClientProfileType {
       id
+      age
```

Comment:
Hey @mikefeldberg 

2 things I noticed.

1. Is the Last Location coming later on?
2. The HMIS ID is not matching on the card and profile (personal Info). 

<img width="360" src="https://github.com/user-attachments/assets/8912c020-acbb-412a-a840-0b7e64699268" />
<img width="360" src="https://github.com/user-attachments/assets/f17e72a6-9923-4b99-902c-10a3464cbfeb" />

Reply:
1. yes. Residence address will be added in DEV-166. Conditional location of last interaction will be added in DEV-760. 
2. fixed

Reply:
I see your fix now. :)